actions/banked-months-updater.vbs

```diff
@@ -270,34 +273,36 @@ For i = 0 to Ubound(banked_months_array, 2)
 	banked_months_array(banked_mo_count_const	, i) = banked_months_count
 	banked_months_array(banked_mo_string_const	, i) = banked_months_string
 	'ABAWD Determinations
-	If abawd_counted_months = 3 then 
+	If abawd_counted_months => 3 then 
 		banked_months_array(used_all_abawd_mo_const, i) = True 
 		banked_months_array(abawd_month_eval_const, i) = "All ABAWD/TLR months used."
 	Elseif abawd_counted_months < 3 then 
 		banked_months_array(used_all_abawd_mo_const, i) = False
-		banked_months_array(abawd_month_eval_const, i) = "Only " & abawd_counted_months & " have been used. This member does not require banked months yet."
-	Else 
-		banked_months_array(member_in_error_const, i) = banked_months_array(member_number_const, i) & " " & banked_months_array(member_first_name_const, i) & " has used " & abawd_months_count & " abawd months. Only 3 are allowed. Updates are needed to this STAT/WREG and/or ABAWD Tracking Record before the script can support this case." 
-		banked_months_array(abawd_month_eval_const, i) = abawd_counted_months & " have been used. Only 3 are allowed. Updates are needed."
+		banked_months_array(abawd_month_eval_const, i) = "Only " & abawd_counted_months & " have been used."
+        banked_months_array(member_in_error_const, i) = banked_months_array(member_number_const, i) & " " & banked_months_array(member_first_name_const, i) & " has used " & abawd_months_count & " abawd months. This member does not require banked months yet."
+	'Else 
```

Comment:
Thinking abawd_months_count should be abawd_counted_months

Reply:
Good catch!